### DISCUSSION TITLE
has any function or hooks to get all global authors in a custom component?

### DISCUSSION THREAD

**Discussion by pincman** [facebook/docusaurus#7011] [Q&A] [answered]
has any function or hooks to get all global authors in a custom component?

**Comment by slorber** [comment, marked as answer]
no, but you can implement a plugin to read any yml/json file and expose it through global data api

you can as well read those files in config and provide data globally with `config.customFields`: https://docusaurus.io/docs/api/docusaurus-config#customfields
